common/api/core-frontend.api.md

```diff
@@ -3764,9 +3764,15 @@ export function getCesiumAssetUrl(osmAssetId: number, requestKey: string): strin
 // @public
 export function getCompressedJpegFromCanvas(canvas: HTMLCanvasElement, maxBytes?: number, minCompressionQuality?: number): string | undefined;
 
+// @internal
+export function getCopyrights(vp: ScreenViewport): Map<string, number>;
+
 // @internal (undocumented)
 export function getFrustumPlaneIntersectionDepthRange(frustum: Frustum, plane: Plane3dByOriginAndUnitNormal): Range1d;
 
+// @beta
+export function getGooglePhotorealistic3DTilesURL(): string;

```

Comment:
nit: inconsistent naming. "GooglePhotorealistic3DTiles" here, "GP3DT" elsewhere. The former is a mouthful, the latter is cryptic. IMO somebody searching our docs for Google 3d tile support should be able to find it by typing "google", which won't work for GP3DT.
I suggest using "Google3DTiles" everywhere.